tripy/tests/test_function_registry.py

```diff
@@ -61,31 +61,30 @@ def test_overloading_string_annotations(self, int_float_registry):
         assert int_float_registry["transform"](1.0) == 0.0
 
     def test_error_on_missing_overload(self, int_float_registry):
+        # Note presence of ANSI color codes. Also note that the last | has a space after it
```

Comment:
Why was this working before without the ANSI color codes? 
We have a `ANSI_ESCAPE` regex in `helper.py`. Maybe we can update `helper.raises` to strip out color codes?

Reply:
I think it was passing spuriously before (see the above comment about the pipe characters, I think they would have matched _anything_).

Reply:
There's an issue with using `ANSI_ESCAPE` in `helper.raises`, namely it is relying on `pytest.raises` to do the matching. I am not sure it will be possible to replace the ANSI sequences at that stage, but I'll try it.

Reply:
Sure, let's leave it as it is for now then. If color codes become too painful later we can implement our own matching logic.